### PR TITLE
chore: mark id as a fail when appends requests is failed

### PR DESCRIPTION
**Pull request by ono-max**

### CODE REVIEW

File: pkg/storage/v2/bigquery/writer/client.go [hunk 1]
```diff
@@ -101,41 +93,41 @@ func NewWriter(
 
 func (w *writer) AppendRows(
 	ctx context.Context,
-	msgs [][]byte,
-) error {
+	batches [][][]byte,
+) ([]int, error) {
+	fails := make([]int, len(batches))
```
Comment (masaaania) [inline pkg/storage/v2/bigquery/writer/client.go:98]: Shouldn't this be `make([]int, 0, len(batches))` ?

**Review by masaaania — APPROVED** [review]
lgtm!

**Comment by ono-max** [comment]
@cre8ivejp 

Could you review this PR? Thx.

**Review by cre8ivejp — APPROVED** [review]
Thank you!
nit: chore: mark event id as failed when append requests fail

**Pull request merged**